optimum/exporters/tasks.py

```diff
@@ -558,7 +558,7 @@ def get_supported_tasks_for_model_type(
                 The model type to retrieve the supported tasks for.
             exporter (`str`):
                 The name of the exporter.
-            model_name (`str`, *optional*):
+            model_name (`Optional[str]`, *optional*):
```

Comment:
Let's not spend too much time on this, but I personally don't think it's good to put `Optional` in docstrings.

For example this is clearer to me:
https://huggingface.co/docs/transformers/v4.24.0/en/main_classes/trainer#transformers.Trainer.tokenizer
https://github.com/huggingface/transformers/blob/94b3f544a1f5e04b78d87a2ae32a7ac252e22e31/src/transformers/trainer.py#L301

Reply:
Yes, but those are not synonyms.

Reply:
This is only confusing for the people reading the doc.